src/utils/status/v2vVMware/v2vVMwareStatus.js

```diff
@@ -28,24 +28,29 @@ const phaseMapper = {
 const hasV2vVMWareStatus = v2vvmware => {
   const status = phaseMapper[getStatusPhase(v2vvmware)];
 
-  return status ? { status } : NOT_HANDLED;
-};
+  if (status) {
+    return { status };
+  }
 
-const hasSetStatus = flags => {
-  if (flags.isConnecting) {
-    return V2V_WMWARE_STATUS_CONNECTING;
+  if (v2vvmware) {
```

Comment:
`v2vvmware` does not have initial status so we guess `V2V_WMWARE_STATUS_CONNECTING` until we get the correct reported status